OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationExtender.java

```diff
@@ -133,34 +133,42 @@ OSNotificationDisplayedResult displayNotification() {
     * <br/><br/>
     * @see OSNotificationReceived#complete()
     */
-   void processNotification() {
-      // If the developer did not call display from notificationProcessing handler
-      if (!developerProcessed) {
-
+   void processNotification(boolean internalFlow) {
```

Comment:
The `boolean` `internalComplete` was added to `OSNotificationReceived` in this PR. I think we should use this name here to keep it consistent.